deployments/helm/KubeArmorOperator/crds/operator.kubearmor.com_kubearmorconfigs.yaml

```
@@ -57,6 +57,12 @@ spec:
                 type: string
               defaultVisibility:
                 type: string
+              enableStdOutAlerts:
```

NIT: Just to keep consistency with KubeArmor's helm chart.
```suggestion
              enableStdoutAlerts:
```